darc/darc.go

```diff
@@ -937,6 +953,8 @@ func (id Identity) Verify(msg, sig []byte) error {
 		return id.Proxy.Verify(msg, sig)
 	case 4:
 		return id.EvmContract.Verify(msg, sig)
+	case 6:
+		return id.TSM.Verify(msg, sig)
```

Comment:
Isn't `case 5:` missing?

Reply:
The DID identity didn't get implemented completely, so it doesn't have a verification.